adapters/pangle/pangle.go

```diff
@@ -0,0 +1,201 @@
+package pangle
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type pangleAdapter struct {
```

Comment:
There's no reason to repeat the package name in this type. The name `adapter` will be fine.

Reply:
fixed